packages/block-editor/src/components/spacing-sizes-control/index.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { __experimentalText as Text } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import AllInputControl from './all-input-control';
+import InputControls from './input-controls';
+import AxialInputControls from './axial-input-controls';
+import LinkedButton from './linked-button';
+import { DEFAULT_VALUES, isValuesMixed, isValuesDefined } from './utils';
+import useSetting from '../use-setting';
+
+export default function SpacingSizesControl( {
+	inputProps,
+	onChange,
+	label = __( 'Spacing Control' ),
+	values,
+	sides,
+	splitOnAxis = false,
+	useSelect,
+	minimumCustomValue = 0,
+} ) {
+	const spacingSizes = [
+		{ name: 0, slug: '0', size: 0 },
+		...useSetting( 'spacing.spacingSizes' ),
```

Comment:
This appears to throw a fatal in the site editor or a block error if spacing sizes are disabled or not available. What should happen for this component if there are no presets? I was wondering if we'd default to just the custom sizes view?

| Block editor | Site editor |
| --- | --- |
| <img width="693" alt="image" src="https://user-images.githubusercontent.com/14988353/184262976-8f87d332-00a8-4deb-b705-56ad927cdcb9.png"> | <img width="1277" alt="image" src="https://user-images.githubusercontent.com/14988353/184262992-9a40d6e3-481f-4a40-a01d-c7c9581b3525.png"> |

(I noticed this because I forgot to update my `theme.json` file after testing out https://github.com/WordPress/gutenberg/pull/43105 🙂)

Reply:
nice catch, let me think about that